client/actions/main.ts

```diff
@@ -68,6 +73,7 @@ import eventsPlanningUi from './eventsPlanning/ui';
 import * as selectors from '../selectors';
 import {validateItem} from '../validators';
 import {searchParamsToOld} from '../utils/search';
+import {searchRawAndStore} from '../api/search';
```

Comment:
Rather than import the function directly, please use the `planningApi` entry point (which is already [imported](https://github.com/superdesk/superdesk-planning/blob/develop/client/actions/main.ts#L6))

Then you can simply use:
`planningApi.search.searchRawAndStore(...`)